apps/web-giddh/src/app/settings/integration/setting.integration.component.ts

```diff
@@ -859,20 +858,29 @@ export class SettingIntegrationComponent implements OnInit, AfterViewInit {
         }
     }
 
-    // public selectedOtpType(event: IOption, index: number) {
-    //     console.log(event, index);
-    //     // if(event && this.paymentFormObj && this.paymentFormObj.userAmountRanges && event.value === 'BANK' ) {
-    //     // this.paymentFormObj.userAmountRanges[index].approvalUniqueName = '';
-
-    //     // }
-    // }
+    /**
+     * To reset aaproval name if OTP type selected
+     *
+     * @param {IOption} event OTP type option event
+     * @param {number} index index number
+     * @param {boolean} isUpdate True if update mode
+     * @memberof SettingIntegrationComponent
+     */
+    public selectedOtpType(event: IOption, index: number, isUpdate: boolean, parentIndex?: number): void {
+        if (event && this.paymentFormObj && this.paymentFormObj.userAmountRanges  && !isUpdate) {
+            this.paymentFormObj.userAmountRanges[index].approvalUniqueName = '';
+        } else if (event && this.registeredAccount && this.registeredAccount[parentIndex].userAmountRanges && isUpdate) {
```

Comment:
Consider adding nullity check to `this.registeredAccount[parentIndex]`

Reply:
Already done and its defined object no need it